src/libraries/System.Windows.Extensions/src/System/Media/SoundPlayer.cs

```diff
@@ -50,7 +51,7 @@ public SoundPlayer()
             _loadAsyncOperationCompleted = new SendOrPostCallback(LoadAsyncOperationCompleted);
         }
 
-        public SoundPlayer(string soundLocation) : this()
+        public SoundPlayer(string? soundLocation) : this()
         {
             SetupSoundLocation(soundLocation ?? string.Empty);
         }
```

Comment:
`public SoundPlayer(Stream stream)` below should be `public SoundPlayer(Stream? stream)`